lib/ex_backend_web/controllers/s3_controller.ex

```diff
@@ -1,7 +1,7 @@
 defmodule ExBackendWeb.S3Controller do
   use ExBackendWeb, :controller
 
-  # import ExBackendWeb.Authorize
+  import ExBackendWeb.Authorize
```

Comment:
I searched for authorize module file, it is located inside the controllers folder, is it meaningful to place it in auth?